r6sapi/definitions/loadouts.py

```diff
@@ -0,0 +1,1079 @@
+"""
```

Comment:
I'm not opposed to having the definitions hard coded, but am conscious it results in the library being more sensitive to API / game changes (updates etc.), increasing the effort required to keep it up to date, whereas before I was trying to move in the opposite direction.

I think for now this solution is fine, but I think a solution where we try and fetch & cache definitions from online first, and if it fails we fallback on these might be better. It might not be worth it though.

Reply:
Yep that is a major downside of having the definitions hard coded; However I cannot find any authoritative sources of this information (please ubi make your api public).  The definitions package is intentionally simple so if we end up doing that down the line, we can simply tweak the converters.py to use online -> hardcoded